### ISSUE TITLE
Note: jasmine2 does not yet support per-spec timeouts

### ISSUE THREAD

**Issue by juliemr** [angular/protractor#1701]
In Jasmine 1.3, you can do:

```js
it('does stuff', function() {
  ...
}, 100);
```
To specify a timeout of 100 ms.

Jasmine2 has added this feature in master (https://github.com/jasmine/jasmine/commit/a4faa80be464b1054293ac3179eb21b2ccc8c472) but it is NOT YET released in version 2.1.1.

This causes `npm test` to fail for Protractor's own test because our timeout suite (expected to fail) instead passes.

TODO - close this issue when a new version of Jasmine is released.

**Comment by juliemr** [comment]
Done, now that we've upgraded to Jasmine 2.3.

**Comment by prabhu-telsiz** [comment]
@juliemr 

Im using 
"jasmine": "2.3.1"
"Protractor: Version 2.1.0"

it('does stuff', function() {
  ...
}, 1000);

I tried this and I got this error.

Jasmine spec timed out. Resetting the WebDriver Control Flow.
" Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."

But the same spec used to work before updating protractor and Jasmine.
